### PR TITLE
Minor Flux Editor Polish

### PR DESCRIPTION
**Pull request by alexpaxton**
- Make `Threesizer` handles/headers not look janky when in horizontal mode
- Match background color of script panel with its scrollbars
- Style `yield` panel name in builder

screen shot 2018-06-13 at 4 12 35 pm

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: ui/src/flux/components/TimeMachineVis.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ class TimeMachineVis extends PureComponent<Props, State> {
             currentView={visType}
             onToggleView={this.selectVisType}
           />
-          <span>{yieldName}</span>
+          <div className="yield-node--name">{`"${yieldName}"`}</div>
```
Comment (ischolten) [inline ui/src/flux/components/TimeMachineVis.tsx]: nice!

File: ui/src/shared/components/threesizer/Division.tsx [hunk 2]
```diff
@@ -67,38 +67,41 @@ class Division extends PureComponent<Props> {
...
-          style={this.handleStyle}
-          title={this.title}
-          draggable={draggable}
-          onDragStart={this.drag}
-          className={this.handleClass}
-          onDoubleClick={this.handleDoubleClick}
-        >
-          <div className={this.titleClass}>{name}</div>
-        </div>
+        {this.renderDragHandle}
```
Comment (ischolten) [inline ui/src/shared/components/threesizer/Division.tsx]: nice! be the change!

File: ui/src/shared/components/threesizer/Division.tsx [hunk 3]
```diff
@@ -123,6 +126,53 @@ class Division extends PureComponent<Props> {
...
+      orientation,
+      headerButtons,
+      menuOptions,
+    } = this.props
+
+    if (!name) {
+      return
+    }
+
+    if (orientation === HANDLE_HORIZONTAL && handlePixels >= 20) {
```
Comment (ischolten) [inline ui/src/shared/components/threesizer/Division.tsx]: this `handlePixels >= 20` is used in more than once. how do you feel about making that 20 a constant?

Comment (alexpaxton) [reply ui/src/shared/components/threesizer/Division.tsx]: Done!

File: ui/src/shared/components/threesizer/DivisionHeader.tsx [hunk 4]
```diff
@@ -8,14 +8,20 @@ interface Props {
...
 
 class DivisionHeader extends PureComponent<Props> {
   public render() {
+    const {name} = this.props
+
     return (
       <div className="threesizer--header">
-        {this.props.buttons.map(b => b)}
-        <DivisionMenu menuItems={this.menuItems} />
+        {name && <div className="threesizer--header-name">{name}</div>}
```
[the remark this answers is not in the document]
Comment (alexpaxton) [reply ui/src/shared/components/threesizer/DivisionHeader.tsx]: Done!

**Pull request merged**